### PR TITLE
Adds admonition re: Dataverse packages

### PR DESCRIPTION
**Pull request by sallain**
Connected to archivematica/Issues#269

### CODE REVIEW

File: user-manual/transfer/dataverse.rst [hunk 1]
```diff
@@ -10,20 +10,24 @@ Overview
...
-Datasets are retrieved directly using the Dataverse API and processed like 
-standard transfers, with some additional processing as described below.
+Datasets are retrieved directly using the Dataverse API and processed using the
+Dataverse :ref:`transfer type <transfer-types>`, which enables some additional
+processing steps as described below.
 
-Dataverse integration is supported by Archivematica version 1.8 and above, and
-has been developed and tested using Dataverse version 4.8.6
+Datasets can also be exported from Dataverse and placed in a transfer source
+location that is visible to Archivematica.
```
Comment (ross-spencer) [inline user-manual/transfer/dataverse.rst]: I think we may need to nuance this a little @sallain, Or perhaps leave it undocumented for now. I'm not sure. 

It takes some wrangling at the moment like needing to include an `agents.json` file, and extract Dataverse bundles into their component parts if bundles are used. Basically everything the Storage Service does but manually :frowning_face:

File: user-manual/transfer/dataverse.rst [hunk 2]
```diff
@@ -34,101 +38,112 @@ has been developed and tested using Dataverse version 4.8.6
...
 checksums provided by Dataverse.
 
+.. important::
+
+   When you are processing a Dataverse dataset that includes packaged material
+   (i.e. .zip or .tar files), Archivematica can extract the contents of these
+   files and run preservation microservices on the contents. This occurs during
+   *Microservice: Extract packages* on the :ref:`Transfer <transfer>` tab.
+   However, due to a `known bug`_, you must not delete the packages after they
+   have been extracted.
```
Comment (ross-spencer) [inline user-manual/transfer/dataverse.rst:86]: :+1:

**Review by ross-spencer — COMMENTED** [review]
Just the one query really @sallain around Dataverse transfers without the Storage Service. Looks good otherwise.

File: user-manual/transfer/dataverse.rst (hunk 1 above)
Comment (sallain) [reply user-manual/transfer/dataverse.rst]: Okay, I will remove!

**Comment by sallain** [comment]
This got merged (confirmed in Gitolite and in 1.8 commit history), not sure why it's not showing up here :woman_shrugging:

**Pull request closed without merging**